### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by iamvasuh**
Share is replaced with skunkworks-crow

Closes #

#### Before submitting this PR, please make sure you have:
- [ ] run `./gradlew pmd checkstyle lint findbugs` and confirmed all checks still pass OR confirm CircleCI build passes
- [ ] verified that any code or assets from external sources are properly credited in comments and/or in the [about file](https://github.com/opendatakit/share/blob/master/share_app/src/main/assets/open_source_licenses.html).

### CODE REVIEW

**Comment by shobhitagarwal1612** [comment]
Thanks for submitting the PR. Please fill out the above PR template.

File: README.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
-# Share
+# Skunkworks-crow
```
Comment (shobhitagarwal1612) [inline README.md]: Please make it lower-case `skunkworks-crow`

File: README.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 [![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-[![Build status](https://circleci.com/gh/opendatakit/share.svg?style=shield&circle-token=:circle-token)](https://circleci.com/gh/opendatakit/share)
+[![Build status](https://circleci.com/gh/opendatakit/skunkworks-crow.svg?style=shield&circle-token=:circle-token)](https://circleci.com/gh/opendatakit/skunkworks-crow)
 [![Slack status](http://slack.opendatakit.org/badge.svg)](http://slack.opendatakit.org)
 
 ## Table of Contents
 1. [Introduction](#introduction)
-    1. [About Share](#about-share)
-    2. [Why Share](#why-share)
+    1. [About Skunkworks-crow](#about-skunkworks-crow)
```
Comment (shobhitagarwal1612) [inline README.md]: Same as above

File: README.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
+    2. [Why Skunkworks-crow](#why-skunkworks-crow)
```
Comment (shobhitagarwal1612) [inline README.md]: Same as above

File: README.md [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
 2. [Installation and setup](#installation)
     1. [Install and setting up ODK Collect](#install-collect)
-    2. [Install Share](#install-share)
-3. [Using Share](#using-share)
+    2. [Install Skunkworks-crow](#install-skunkworks-crow)
```
Comment (shobhitagarwal1612) [inline README.md]: Same as above

File: README.md [hunk 5]
```diff
@@ -1,17 +1,17 @@
...
+3. [Using Skunkworks-crow](#using-skunkworks-crow)
```
Comment (shobhitagarwal1612) [inline README.md]: Same as above

**Comment by iamvasuh** [comment]
ok i will just do it

**Comment by shobhitagarwal1612** [comment]
You can push to the same branch and this PR will be updated. You don't need to close the PR

**Pull request closed without merging**